wandb/cli/cli.py

```diff
@@ -1317,13 +1372,20 @@ def agent(ctx, project, entity, count, sweep_id):
     default=None,
     help="The queue to push sweep jobs to.",
 )
+@click.option(
+    "--job",
+    "-j",
+    default=None,
+    help="The name of the job that encapsulates a single run in the sweep.",
```

Comment:
check with kyle on that the proper way to pass this should be. Do we have a concept of a job URI @KyleGoyette ? i imagine hugo needs the entity name, project name, artifact name, and version. something like: `wandb/launch-test/git-repo:v0`

Reply:
Job URIs are identical to artifatc URIs. So the artifact can optionally be prefixed with entity/project but is not necessary. If its not prefixed the assumption will be that the artifact exists in the current entity/project scope